superagi/models/agent_execution_feed.py

```diff
@@ -52,7 +52,7 @@ def get_last_tool_response(cls, session: Session, agent_execution_id: int, tool_
             if agent_execution_feed.feed.startswith("Tool"):
                 return agent_execution_feed.feed
         return ""
-
+    
     @classmethod
```

Comment:
revert back unrelated files too many changes to merge at a time

Reply:
reverted back this change